addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -1772,6 +1772,10 @@ QUnit.module("Views", (hooks) => {
                 trululu(record) {
                     // when trululu changes, push another record in product_ids.
                     // only push a second record once.
+
+                    // The new relational model doesn't send un unchanged field
+                    // LPE To check
+                    record.product_ids = record.product_ids || [];
```

Comment:
please don't do such things, we'll almost certainly not re-read this line again, and we'll merge like this. If there something strange with the test, keep it skipped, or investigate now.